APIs/schemas/edid.json

```diff
@@ -2,16 +2,26 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "required": [
+    "version",
     "manufacturer",
     "manufacture_week",
     "manufacture_year",
-    "version",
     "screen_size",
     "gamma",
     "display_type",
     "established_timings"
   ],
   "properties": {
+    "version": {
+      "description": "String formatted TAI timestamp (<seconds>:<nanoseconds>) indicating precisely when an attribute of the resource last changed",
+      "type": "string",
+      "pattern": "^[0-9]+:[0-9]+$"
+    },
+    "edid_binary_href": {
+      "description": "HTTP(S) accessible URL to a binary EDID file.",
+      "type": ["string", "null"],
+      "format": "uri"
+    },
```

Comment:
Do we need this if it's in the API?

Reply:
I removed it from the API in the last commit. I added schema for Sink based on Resource to label Sinks and track versions. Now Sink consists of Resource Core and EDID which seems logically to me. I think it may be implemented similarly to "manifest_href" of Senders instead of making a separate REST API resource for this in RAML.

Reply:
Resolving it since /edid endpoint turned back.